spotifyconnector/connector.py

```diff
@@ -392,9 +410,9 @@ def episodes(
         end: Optional[dt.date] = None,
         page: int = 1,
         size: int = 50,
-        sortBy: str = "releaseDate",
-        sortOrder: str = "descending",
-        filter: str = "",
+        sort_by: str = "releaseDate",
+        sort_order: str = "descending",
+        filter_by: str = "",
```

Comment:
This is a breaking change, but I've checked and we don't use these parameters in our pipelines, so it's okay to change it.